src/controller/Controller.h

```diff
@@ -11,10 +11,12 @@
 
 #define EXTENDED_SCANCODE_BIT (1 << 8)
 #define AXIS_SCANCODE_BIT (1 << 9)
+#define MAX_AXIS_RANGE 85.0
```

Comment:
```suggestion
#define MAX_AXIS_RANGE 85.0f
```
We use floats in most places and not doubles.